src/components/StyledFieldset.jsx

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components';
+
+const FieldsetStyles = styled.fieldset`
+  width: 50vw;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+
+  legend {
+    padding: 3rem 0 1rem 0;
+    font-size: 3rem;
+    font-weight: 700;
+  }
+  p {
+    font-size: 1rem;
+  }
+
+  p.instructions {
+    font-size: 1.6rem;
+    font-weight: 700;
+    margin-top: 2rem;
+  }
+
+  label {
+    font-size: 1.5rem;
+    font-weight: 600;
+    margin-bottom: 1rem;
+  }
+
+  .skillLabel {
+    font-size: 1rem;
+    font-weight: 400;
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
I'm gonna call Joseph out and say this was a mistake.
Can we just use the same font we use everywhere else?

Good eye for detail though